### DISCUSSION TITLE
Allow wildcard * in filter entities in /developer-tools/state

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#7694] [Dashboard] [answered]
as title,
would be very nice to be able to use a wildcard. to be able to eg select all device_trackers with the _bt in the slug.

Right now we can only use either, not both

(sorry if this is not the correct category, tried 'other' but that wouldn't give me the green Start discussion button)

**Comment by spacegaier** [comment, marked as answer]
I just added that to my open PR https://github.com/home-assistant/frontend/pull/7582 since that was already changing the filter code anyway. Can you check if that is what you had in mind? I added two screenshots showing the new behavior.

An alternative could be to re-use the fuzzy search logic that the quick bar is using.

**Reply by Mariusthvdb** [reply]
Yes! I think it is, great! Thx

Marked as answer ;-)
